Source/Core/Core/HW/EXI/BBA/BuiltIn.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+#ifdef _WIN32
+#include <WinSock2.h>
+#else
+#include <netinet/in.h>
+#endif
```

Comment:
Are these needed in the header? I see no obvious place they're used, so if you can please move them to the cpp file.

Reply:
sockaddr_in from;
  sockaddr_in to;
are using it, unless you have a commone one that redefine them?

Reply:
Ah, my bad then.